tests/integration/test_api.py

```diff
@@ -20,9 +21,11 @@ def is_valid_uuid(u):
     except ValueError:
         return False
 
-class APITestCase(ServerTestCase):
+class APITestCase(ServerTestCase, DatabaseTestCase):
```

Comment:
Does it make sense to make an IntegrationTestCase which makes this easy as we make more tests?